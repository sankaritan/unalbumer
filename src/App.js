import React, { Component } from "react";
// import { Route, Link } from "react-router-dom";
import "./App.css";
import Root from "./components/Root";

class App extends Component {
  render() {
    return (
      <div className="App">
        {/* <header className="App-header">
          <Link to="/">Home</Link>
        </header> */}
        <div>
          <Root />
          {/* <Route exact path="/" component={Home} /> */}
        </div>
      </div>
    );
  }
}

export default App;
