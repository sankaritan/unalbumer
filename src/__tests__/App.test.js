import React from "react";
import { App } from "../App";
import Adapter from "enzyme-adapter-react-16";
import Enzyme, { shallow } from "enzyme";

Enzyme.configure({ adapter: new Adapter() });

describe("App wrapper component", () => {
  it("App renders correctly", () => {
    const tree = shallow(<App />).debug();
    expect(tree).toMatchSnapshot();
  });
});
