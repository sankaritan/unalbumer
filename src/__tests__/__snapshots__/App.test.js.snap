// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App wrapper component App renders correctly 1`] = `
"<div className=\\"App\\">
  <div>
    <Connect(Root) />
  </div>
</div>"
`;
